scripts/ensure-env.js

```diff
@@ -0,0 +1,7 @@
+var fs = require("fs");
+var env = require("path").join(__dirname, "../", ".env");
```

Comment:
`join(__dirname, "..", ".env");` (path takes care of putting the correct delimiter in between the various path segments)